File: desktop/src/com/heroes3/livewallpaper/desktop/DesktopLauncher.java
package com.heroes3.livewallpaper.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;

import com.heroes3.livewallpaper.clojure.LiveWallpaperEngine;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.height = 1136;
		config.width = 640;

		new LwjglApplication(new LiveWallpaperEngine(), config);
	}
}

File: desktop/src/com/homm3/livewallpaper/desktop/DesktopLauncher.kt
package com.homm3.livewallpaper.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration().apply {
            backgroundFPS = 60
            foregroundFPS = 60
            x = 0
            y = 0
            height = 426
            width = 500
        }

        LwjglApplication(DesktopEngine(), config)
    }
}